src/collections/_documentation/platforms/php/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: PHP
+sidebar_order: 3
+---
+
+{% include learn-sdk.md platform="php" %}
+
+The [Sentry PHP SDK](https://packagist.org/packages/sentry/sentry) provides
+support for PHP 7.1 or later.
+
+This documentation goes over some PHP specific things such as integrations to
+frameworks.
+
+## Integrations
+
+*Integrations* extend the functionality of the SDK for some common frameworks and
+libraries.  They can be seen as plugins that extend the functionality of the Sentry SDK.  
+Integrations are configured by a call to `init()`.
+
+## PHP Specific Options
+
+PHP provides some additional options, all options can be passed into `init()`.
+
+#### send_attempts
+
+The number of attempts that should be made to send an event before erroring
+and dropping it from the queue.
+By default this option is set to `6`.
+
+#### project_root
+
+The root of the project source code. As Sentry is able to distinguish project
+files from third-parties ones (e.g. vendors), this option can be configured to
+mark the directory containing all the source code of the application.
+
+For example, assuming that the directory structure shown below exists, marking
+the project root as ``project-folder/src/`` means that every file inside that
+directory that is part of a stacktrace frame will be marked as "application
+code".
+
+```
+
+    project-folder/
+    ├── vendor/
+        ├── foo/
+    ├── src/
+        ├── bar/ <-- these are going to be marked as application files
+
+``` 
+
+#### enable_compression
+
+If this option is enabled, `gzip` compression will be enabled. Default is `true`.
+
+#### context_lines
+
+This option sets the number of lines of code context to capture. If `null` is
+set as the value, no source code lines will be added to each stacktrace frame.
+By default this option is set to `3`.
+
+#### excluded_exceptions
+
+Sometimes you may want to skip capturing certain exceptions. This option sets
+the FQCN of the classes of the exceptions that you don't want to capture. The
+check is done using the `instanceof` operator against each item of the array
+and if at least one of them passes the event will be discarded.
+
+#### excluded_app_paths
+
+This option configures the list of paths to exclude from the `app_path` detection.
+
+#### prefixes
+
+This option sets the list of prefixes which should be stripped from the filenames
+to create relative paths.
+
+## Transport
+
+Transports are the classes in Sentry PHP that are responsible for communicating
```

Comment:
I believe we should note that once a client is initialized the transport cannot be changed anymore on that instance